performant-software/performant-web
# Created a new projects index page

## What does this PR do

- Updated global nav file **front-header.html** with link to projects.html
- Updated **front-head.html** to include "Cube Portfolio" plugin (css and JS)
- Included the init "Cube portfolio" script in front-scripts.html
- Installed all necessary "Cube portfolio" decencies on "_theme" folder
- Fixes #31 

## How to test this

- Navigate to projects tab and there should be a list of 4 projects including place holder categories
